lib/node_modules/@stdlib/math/base/special/ceiln/README.md

```diff
@@ -98,6 +98,101 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/ceiln.h"
+```
+
+#### stdlib_base_ceiln( x, n )
+
+Rounds a `numeric` value to the nearest multiple of `10^n` toward positive infinity.
+
+```c
+// Round a value to 2 decimal places:
+double y = stdlib_base_ceiln( 3.141592653589793, -2 );
+// returns 3.15
+
+// If n = 0, `ceiln` behaves like `ceil`:
+double y = stdlib_base_ceiln( 3.141592653589793, 0 );
+// returns 4.0
+
+// Round a value to the nearest thousand:
+double y = stdlib_base_ceiln( 12368.0, 3 );
+// returns 13000.0
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+-   **n**: `[in] double` integer power of 10.
+
+```c
+double stdlib_base_ceiln( const double x, const double n );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/ceiln.h"
+#include <stdio.h>
+
+int main() {
+    double x[] = { 3.14, -3.14, 0.0, 0.0/0.0 };
+
+    double y;
+    int i;
+    for ( i = 0; i < 4; i++ ) {
+        y = stdlib_base_ceiln( x[ i ], 0 );
```

Comment:
This example is not super illustrative, as `ceiln` behaves like `ceil` when `n = 0`.